.github/workflows/main.yml

```diff
@@ -91,20 +93,21 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         working-directory: setuptools/tests/config
         run: python -m downloads.preload setupcfg_examples.txt
+      - name: Workaround for unreleased PyNaCl (pyca/pynacl#805)
+        if: contains(matrix.python, 'pypy')
+        run: echo "SETUPTOOLS_ENFORCE_DEPRECATION=0" >> $GITHUB_ENV
+      - name: Install tox
+        run: python -m pip install tox
       - name: Pre-build distributions for test
         shell: bash
         run: |
           rm -rf dist
           # workaround for pypa/setuptools#4333
+          tox -e version
```

Comment:
Surely this new command isn't meant to be a workaround for the mentioned issue; probably the command should appear before that comment. It would be nice if this command didn't need to happen at all (if it happened as part of running tox / installing the source tree).

Reply:
I have changed the order of the lines so that this problem does not happen.

In theory we could remove it, as it would not be a problem for the tests, but I think it is nice to have a consistent naming of the tested artifacts, so for now, I prefer to leave it there (which might change after more investigation).